src/shared_utilities/drivers.jl

```diff
@@ -41,15 +41,21 @@ Container for holding prescribed atmospheric drivers and other
 information needed for computing turbulent surface fluxes when
 driving land models in standalone mode.
 
-The arguments labeled with "function of time" are typically either of type
-`Function` or type `Spline1D` from the `Dierckx.jl` package.
-
 Since not all models require co2 concentration, the default for that
 is `nothing`.
 $(DocStringExtensions.FIELDS)
 """
-struct PrescribedAtmosphere{FT, LP, SP, TA, UA, QA, RA, CA, DT} <:
-       AbstractAtmosphericDrivers{FT}
+struct PrescribedAtmosphere{
+    FT,
+    LP <: Union{Nothing, AbstractTimeVaryingInput},
```

Comment:
When does the Nothing case arise?

Reply:
I found it here:
https://github.com/CliMA/ClimaLSM.jl/blob/058a66b39652cf72477fbcd174c5fbb4066d8703/test/shared_utilities/drivers.jl#L8-L17

But if we don't need it, I will remove support for `Nothing`

Reply:
let me double check.

Reply:
we can keep it for now and Ill open an issue? no need to go down a rabbit hole for something tangential.

Reply:
Open #461